website/daily-snapshots

```diff
@@ -21,7 +21,11 @@ git checkout QA_5_2
 git reset --hard
 # Force an update
 git pull --ff-only
-./scripts/create-release.sh --daily 5.2+snapshot QA_5_2
+if [ -f ./scripts/create-release.sh ]; then
+  ./scripts/create-release.sh --daily 5.2+snapshot QA_5_2
+else
+  ./bin/internal/create-release.sh --daily 5.2+snapshot QA_5_2
```

Comment:
I'm not sure if this is necessary, as the QA_5_2 branch is checked out a few lines earlier.

Reply:
Thanks, good point. I reverted this and added a comment.